server/memes/memeController.js

```diff
@@ -0,0 +1,72 @@
+var http = require('http');
+var querystring = require('querystring');
+
+
+module.exports = {
+
+  getMemes: function(request, response, next){
+    var options = {
+      host: 'version1.api.memegenerator.net',
+      path: '/Instances_Select_ByPopular?languageCode=en',
+      method: 'GET'
+    };
+    return http.get(options, function(res){
+      var body = '';
+      res.on('data', function(chunk){
+        body += chunk;
+      });
+      res.on('end', function(){
+        response.status(200).send(body);
+      });
+    });
+  },
+
+  createMeme: function(request, response, next){
+
+    var postData = querystring.stringify({
+      'username': 'memesformankind',
+      'password': 'memesformankind',
+      'generatorID': request.generator_id || 45,
+      'languageCode': 'en',
+      'imageID': request.imageID || 20,
+      'text0': request.text0 || 'toptext',
+      'text1': request.text1 || 'bottomtext'
```

Comment:
Will need to add bodyParser to get these properties. They'll then be on request.body.PROPERTY.

Also, the key names could be more intuitive